src/cmd/src/lib.rs

```diff
@@ -42,22 +42,22 @@ risingwave_expr_impl::enable!();
 // Entry point functions.
 
 pub fn compute(opts: ComputeNodeOpts) {
-    init_risingwave_logger(LoggerSettings::new("compute"));
```

Comment:
The Logger's name was removed here? Would it become confusing especially for standalone mode?

Reply:
`from_opts` will call `Self::new(O::name())`, so we'll still get the name.

BTW, the name is singleton and only used by distributed tracing as well, which means that we cannot have different names for different modules in standalone deployment. May refer to #13988 for discussions.

Reply:
`from_opts` will still call `new` with the name.

BTW, names are singleton in a process and used by distributed tracing only. May refer to #13988 for discussions about logs under standalone deployments.